augur/clades.py

```diff
@@ -263,6 +263,8 @@ def register_parser(parent_subparsers):
     parser.add_argument('--reference', nargs='+', help='fasta files containing reference and tip nucleotide and/or amino-acid sequences ')
     parser.add_argument('--clades', type=str, help='TSV file containing clade definitions by amino-acid')
     parser.add_argument('--output-node-data', type=str, help='name of JSON file to save clade assignments to')
+    parser.add_argument('--membership-name', type=str, default="clade_membership", help='Key to store clade membership under; use "None" to not export this')
+    parser.add_argument('--label-name', type=str, default="clade", help='Key to store clade labels under; use "None" to not export this')
```

Comment:
> I've chosen the arguments --membership-name and --label-name. I opted to use separate ones because (i) it's super clear what's going on and (ii) it allows to not export one of them if you don't want to.

Just curious, what's the use case to not export membership or label name?

Reply:
I'll admit it's rare, but I have written a few scripts during the pandemic to label certain mutations etc without wanting the corresponding coloring. It's not a huge deal, but I felt there was very little downside to allowing this and the upside is avoiding extra scripts / rules in pipelines.

Reply:
Yeah, and it's not obvious that it would! But any key-value pair [1] in a node-data JSON will be exported as a coloring, and there's no way to stop it. The thinking at the time this design decision was made was that any data in a node-data JSON has been explicitly generated by the user so they will want it exported, but it has resulted in a bit of confusion I think.

[1] well, not any... there are plenty of exceptions!